static/js/jira-server-url.js

```diff
@@ -68,18 +68,50 @@ const requestFailed = () => {
 	$("#gheServerBtnSpinner").hide();
 };
 
+const gheServerUrlErrors = {
+	errorCode: {
+		GHE_ERROR_1: {
+			title: "Invalid URL",
+			message: "That URL doesn't look right. Please check and try again.",
+		},
+		GHE_ERROR_2: {
+			title: "We couldn't verify this URL",
+			message: "Please make sure you've entered the correct URL and check that you've properly configured the hole in your firewall.",
+		},
+		GHE_ERROR_3: {
+			title: "Request failed",
+			message: "We weren't able to complete your request. Please try again."
+		},
+		GHE_ERROR_4: {
+			title: "Something went wrong",
+			message: "We ran into a hiccup while verifying your details. Please try again later."
+		}
+	}
+};
+
+const handleGheUrlRequestErrors = (err) => {
+	requestFailed();
+	$(".jiraServerUrl__validationError").show();
+	$(".errorMessageBox__title").append(title);
```

Comment:
Will `append()` append multi error titles to same dom element if user click the submit button twice and failed twice? Ignore this if it doesn't.